database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
+      // overallRating needs future refactor to generate dynamic rating
+      overallRating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      // totalReviews needs future refactor to generate dynamic count
+      totalReviews: Math.floor(Math.random() * Math.floor(60)) + 10,
+      averageCost: Math.floor(Math.random() * Math.floor(4)) + 1,
+      businessType: `${faker.company.catchPhraseAdjective()}, ${faker.name.jobArea()}`,
+      address1: `${faker.address.streetAddress()}`,
+      address2: `${faker.address.city()}, ${faker.address.stateAbbr()} ${faker.address.zipCode()}`,
+      address3: `b/t ${faker.address.streetName()} ${faker.address.streetSuffix()} & ${faker.address.streetName()} ${faker.address.streetSuffix()}`,
+      address4: `${faker.address.city()}`,
+      phoneNumber: faker.phone.phoneNumber(),
+      url: `${faker.lorem.word()}${faker.internet.domainWord()}.${faker.internet.domainSuffix()}`,
+    };
+    businesses.push(entry);
+  }
+  return businesses;
+};
+
+// generate fake date for 7000 total reviews of the corresponding 100 businesses generated above
+const generateReviews = function () {
+  const reviews = [];
+  for (let i = 1; i < 7000; i += 1) {
+    const entry = {
+      username: faker.internet.userName(),
+      text: faker.lorem.sentences(),
+      rating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      date: new Date(),
+      businessId: Math.floor(Math.random() * Math.floor(100)) + 1,
+    };
+    reviews.push(entry);
+  }
+  return reviews;
+};
+
+const businesses = generateBusiness();
+const reviews = generateReviews();
+
+for (let i = 0; i < businesses.length; i += 1) {
```

Comment:
This for loop and the one below it act to Insert your generate data into your database. Consider moving these two into database/index.js.

This way, your generateData.js file only contains the functions used for generating data. Inserting data can be thought of as a different concern and might fit better in the index.js file. You can export your data generation functions and use them from a different file.

Reply:
That makes sense. I refactored the structure so that my data generation and data insertion operations are defined in separate files. I'm not always great about separation of concerns. It was a great suggestion.